mtags/src/main/scala-3/scala/meta/internal/pc/PcCollector.scala

```diff
@@ -258,6 +258,22 @@ abstract class PcCollector[T](
           .selector(pos.span)
           .map(sym => (symbolAlternatives(sym), sym.sourcePos))
 
+      /* Workaround for missing span in:
+       * class MyIntOut(val value: Int)
+       * object MyIntOut:
+       *   extension (i: MyIntOut) def <<uneven>> = i.value % 2 == 1
+       *
+       * val a = MyIntOut(1).un@@even
+       */
+      case Apply(sel: Select, _) :: _
+          if sel.span.isZeroExtent && sel.symbol.is(Flags.ExtensionMethod) =>
+        untypedPath match
+          case (untypedSel: untpd.Select) :: _ =>
+            Some(
+              symbolAlternatives(sel.symbol),
+              pos.withSpan(untypedSel.nameSpan),
```

Comment:
I think for most cases, the span would be the Apply span, from point to end, so we won't have to use untypedPath, but I have to check how it will work for infix methods